recipes/Semi-ATE-Plugins/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "Semi-ATE-Plugins" %}
+{% set version = "1.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/semi-ate-plugins-{{ version }}.tar.gz
+  sha256: dd1476c92117aa66a9ec2bcfb29bd1b16a8283182ae2772d81ae6b4db71feb53
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - python >=3.8
+    - semi-ate-common ==1.0.6
```

Will this always be kept in sync?
```suggestion
    - semi-ate-common =={{ version }}
```